react.md

```diff
@@ -0,0 +1,137 @@
+# Introduction
+
+[React](https://facebook.github.io/react/) is a JavaScript library for building user interfaces. The following guide is to be followed in addition to the [ES2015](es2015.md) style guide.
+
+# Tooling
+
+To lint our React apps we use:
+
+- [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react)
+
+To plug into [eslint](http://eslint.org).
+
+# Styleguide
+
+General points:
+
+- React Props and States are camelCased
+- Use propTypes where practicable
+- Single line JSX to have parentheses
+- Multiline props when more than 5 props are in a component
+- Multiline props to be in alphabetical order
+- Multiline props to be indented 2 spaces
+```
+<Component
+  AProp={this.handleAProp}
```

Comment:
Should follow your rule on [L17](https://github.com/thefrontiergroup/standards/pull/21/files#diff-67f377003d8e128a03b229dc0a3f87a9R17)